lib/model/model.go

```diff
@@ -173,6 +172,13 @@ func (m *Model) StartDeadlockDetector(timeout time.Duration) {
 // StartFolder constructs the folder service and starts it.
 func (m *Model) StartFolder(folder string) {
 	m.fmut.Lock()
+	folderType := m.startFolderUnlocked(folder)
```

Comment:
I think these methods should be `...Locked` to indicate they must be called in a locked context